cico_run_pylint.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -ex
+
+prep() {
+    yum -y update
+    yum -y install epel-release
+    yum -y install python36 python36-virtualenv which
+}
+
+check_python_version() {
+    python3 tools/check_python_version.py 3 6
+}
+
+# this script is copied by CI, we don't need it
+rm -f env-toolkit
+
+prep
+check_python_version
+./qa/detect-common-errors.sh
```

Comment:
You have not added any of these scripts.

Reply:
He has added all these script as part of previous PR.

Reply:
yes all qa scripts are part of earlier PR - https://github.com/kubesecurity/osa-api-server/pull/8